# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../qiv/00-config.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./Makefile.orig	2004-05-22 03:21:47.000000000 -0400
+++ ./Makefile	2005-02-11 19:03:45.689653447 -0300
@@ -4,7 +4,7 @@
 #######################################################################
 
 # Directory where qiv will be installed under.
-PREFIX = /usr/local
+PREFIX = /usr
 
 # Font to use for statusbar in fullscreen mode
 STATUSBAR_FONT = "fixed"
@@ -14,10 +14,10 @@
 CURSOR = 84
 
 # Should image be centered on screen by default? 1=yes, 0=no.
-CENTER = 1
+CENTER = 0
 
 # Should images be filtered by extension? 1=yes, 0=no.
-FILTER = 1
+FILTER = 0
 
 # This sets the file extentions to filter on (other file types will be
 # skipped.) It should reflect whatever is compiled into imlib.
@@ -28,7 +28,7 @@
 
 # This program will be run on the manual page after it is installed.
 # If you don't want to compress the manpage, change it to 'true'.
-COMPRESS_PROG = gzip -9f
+COMPRESS_PROG = true
 
 # Comment this line out if your system doesn't have libXinerama
 # installed (for centering on dual-screen)
@@ -126,11 +126,6 @@
 	@echo "Installing QIV..."
 	install -s -m 0755 $(PROGRAM) $(PREFIX)/bin
 	install -m 0644 $(PROGRAM).1 $(PREFIX)/man/man1
-	$(COMPRESS_PROG) $(PREFIX)/man/man1/$(PROGRAM).1
-	@if ./qiv -f ./intro.jpg ; \
-	then echo "-- Test Passed --" ; \
-	else echo "-- Test Failed --" ; \
-	fi
 	@echo -ne "\nDont forget to look into the \"qiv-command\" file and install it!\n-> cp qiv-command.example /usr/local/bin/qiv-command\n\n" 
 
 install-xscreensaver: install
